packages/cli/generation/remote-generation/remote-workspace-runner/src/runRemoteGenerationForWorkspace.ts

```diff
@@ -23,20 +23,25 @@ export async function runRemoteGenerationForWorkspace({
 
     const results = await Promise.allSettled(
         generatorGroup.generators.map((generatorInvocation) =>
-            context.runInteractiveTask({ name: generatorInvocation.name }, (interactiveTaskContext) =>
-                runRemoteGenerationForGenerator({
-                    organization,
-                    workspace,
-                    interactiveTaskContext,
-                    generatorInvocation,
-                    version,
-                    audiences: generatorGroup.audiences,
-                })
-            )
+            context.runInteractiveTask({ name: generatorInvocation.name }, async (interactiveTaskContext) => {
+                try {
+                    await runRemoteGenerationForGenerator({
+                        organization,
+                        workspace,
+                        interactiveTaskContext,
+                        generatorInvocation,
+                        version,
+                        audiences: generatorGroup.audiences,
+                    });
+                } catch (e) {
+                    interactiveTaskContext.logger.error("Failed");
+                    interactiveTaskContext.logger.debug(`${e}`);
```

Comment:
This will probably log out `Error`, you should do `interactiveTaskContext.logger.debug(e);`. We have logic here to stringify it correctly https://github.com/fern-api/fern/blob/main/packages/cli/cli/src/cli-context/logErrorMessage.ts

Reply:
<img width="1042" alt="Screen Shot 2022-12-02 at 6 04 09 PM" src="https://user-images.githubusercontent.com/10870189/205417321-36d52f3c-badc-4549-b0e3-2c0fac3280f7.png">